vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -17,8 +17,65 @@ using namespace facebook::react;
 
 namespace Microsoft::JSI {
 
-/*static*/ std::once_flag ChakraRuntime::s_runtimeVersionInitFlag;
-/*static*/ uint64_t ChakraRuntime::s_runtimeVersion = 0;
+namespace {
+
+class HostFunctionProxy {
+ public:
+  HostFunctionProxy(
+      facebook::jsi::HostFunctionType hostFunction,
+      ChakraRuntime &runtime)
+      : m_hostFunction(hostFunction), m_runtime(runtime) {}
```

Comment:
>, [](start = 36, length = 1)

(nit) split initialization into separate lines

Reply:
This was done by clang format.

---
In reply to: [333115061](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333115061) [](ancestors = 333115061)